recipes/mss/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.2.0" %}
+
+package:
+    name: mss
+    version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        # mss import pyproj from basemap and that does not exist in 1.0.8
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
+        - paste
+        - pillow
+        - python-dateutil
+        - pytz
+        - requests
+        - scipy
+        - six
+        - tk 8.5.*
```

Comment:
I missed the line in https://bitbucket.org/wxmetvis/mss/src/44bf88afa47fb81bfc2e02e66d0d2bf8e66d3907/setup.py?fileviewer=file-view-default#setup.py-3

With `setuptools` we can use the `--single-version-externally-managed` and that allows us to remove the run time dependencies from the build time. There is no need for any modification in the code.